tools/Generate_MeanExcitationEnergy.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+#
+# GENERATE MEAN EXCITATION ENERGY DATA FOR Z>18
+# --------
+# This script generates Mean Excitation Energy (MEE) data from
+# semi-empirical approximation for elements with Z>18 and store them in C++ file.
+# This is used in DREAM to complement data from Sauer et al.
+#
+# J. Walkowiak
+# 08.2022
+# #################################################################################
+
+import numpy as np
+import scipy.integrate as integrate
+import math
+import sys
+import pathlib
+import argparse
+
+def GenerateMEE(Z_min=19, Z_max=86):
```

Comment:
Should there be a check at the start of this function to ensure that ``Z_min`` and ``Z_max`` are valid, perhaps?

Reply:
I think for now it can stay as it is. I will probably modify this function in the future, as I plan to work a bit more on this coefficients. Below are main reasons why I don't modify it now:
1. This function is not called by the user, the array it generates is already included in the repository. It can be used if you want to modify the array for some reason, but then you should know what you are doing.
2. Technically it should be valid independent from Z.
3. Z_min and Z_max are in fact defined in https://github.com/chalmersplasmatheory/DREAM/blob/master/include/DREAM/Equations/SlowingDownFrequency.hpp
In theory I can use e.g. cython to link this, but I am afraid that this will generate more problems than it solves.